mathml/relations/html5-tree/math-global-event-handlers.tentative.html

```diff
@@ -28,9 +28,9 @@
 
       for (const name of names) {
         const withoutOn = name.substring(2);
+        const location = MathMLElement.prototype;
```

Comment:
location is used elsewhere with different types, not sure that's a very good name. Also you probably don't need to factor out this variable.